src/test/java/org/elasticsearch/example/realm/CustomRealmTests.java

```diff
@@ -20,12 +20,15 @@
 package org.elasticsearch.example.realm;
 
 import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.util.concurrent.ThreadContext;
+import org.elasticsearch.env.Environment;
 import org.elasticsearch.xpack.security.user.User;
 import org.elasticsearch.xpack.security.authc.RealmConfig;
 import org.elasticsearch.xpack.security.authc.support.SecuredString;
 import org.elasticsearch.xpack.security.authc.support.UsernamePasswordToken;
 import org.elasticsearch.test.ESTestCase;
 
+import static com.carrotsearch.randomizedtesting.RandomizedTest.randomAsciiOfLengthBetween;
```

Comment:
Same comment as `randomAlphaOfLength`, instead of `randomAsciiOfLengthBetween` it should be `randomAlphaOfLengthBetween`.